grpc/src/main/java/com/linecorp/armeria/server/grpc/HandlerRegistry.java

```diff
@@ -209,6 +231,12 @@ Builder addService(String path, ServerServiceDefinition service,
             return this;
         }
 
+        Builder addGlobalExceptionHandler(GrpcExceptionHandlerFunction globalExceptionHandler) {
```

Comment:
How about just `defaultExceptionHandler` because global means something that is applied to across the whole server instance. 
```suggestion
        Builder setDefaultExceptionHandler(GrpcExceptionHandlerFunction defaultExceptionHandler) {
```